src/main/java/contoller/Janggi.java

```diff
@@ -0,0 +1,55 @@
+package contoller;
+
+import static view.InputView.choiceSetUp;
+import static view.InputView.movePointInput;
+import static view.OutputVIew.displayJanggiBoard;
+
+import java.util.List;
+import model.Point;
+import model.Team;
+import model.janggi_board.JanggiBoard;
+import model.janggi_board.JanggiBoardSetUp;
+import view.OutputVIew;
+
+public class Janggi {
+    public void play() {
+        int setUpChoice = choiceSetUp();
+        JanggiBoard janggiBoard;
+        switch (setUpChoice) {
+            case 1:
+                janggiBoard = new JanggiBoard(JanggiBoardSetUp.INNER_ELEPHANT);
+                break;
+            case 2:
+                janggiBoard = new JanggiBoard(JanggiBoardSetUp.OUTER_ELEPHANT);
+                break;
+            case 3:
+                janggiBoard = new JanggiBoard(JanggiBoardSetUp.LEFT_ELEPHANT);
+                break;
+            case 4:
+                janggiBoard = new JanggiBoard(JanggiBoardSetUp.RIGHT_ELEPHANT);
+                break;
+            default:
+                throw new IllegalArgumentException("다시 입력하세요.");
+        }
```

Comment:
`java14` 부터는 `switch` 문을 리턴 값이 있는 식처럼 사용할 수 있습니다.
```suggestion
        JanggiBoard janggiBoard = switch (setUpChoice) {
            case 1 -> new JanggiBoard(JanggiBoardSetUp.INNER_ELEPHANT);
            case 2 -> new JanggiBoard(JanggiBoardSetUp.OUTER_ELEPHANT);
            case 3 -> new JanggiBoard(JanggiBoardSetUp.LEFT_ELEPHANT);
            case 4 -> new JanggiBoard(JanggiBoardSetUp.RIGHT_ELEPHANT);
            default -> throw new IllegalArgumentException("다시 입력하세요.");
        };
```

Reply:
훨씬 깔끔하네요! 수정했습니다!